### PR TITLE
CLM-8: Avoid creating duplicate attributes when adding result for CommonLabTest Order

### PR DESCRIPTION
**Pull request by tahiraniazi**
### Description of what I changed
Avoid creating duplicate attributes when adding result for CommonLabTest Order

Issue I worked on
see https://issues.openmrs.org/browse/CLM-8

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/commonlabtest/LabTest.java [hunk 1]
```diff
@@ -132,8 +132,8 @@ public void setFilePath(String filePath) {
 	 * 
 	 * @param referenceNumber the reference number
 	 * @return {@link LabTest} object(s)
-	 * @deprecated Data provided by this method can be better achieved from 
-	 * appropriate service at point of use.
+	 * @deprecated Data provided by this method can be better achieved from appropriate service at point
```
Comment (seekme94) [inline api/src/main/java/org/openmrs/module/commonlabtest/LabTest.java:135]: Unnecessary change, breaking the formatting

File: api/src/main/java/org/openmrs/module/commonlabtest/LabTest.java [hunk 2]
```diff
@@ -156,8 +156,8 @@ public String getDisplayString() {
 	 * Also @see org.openmrs.Attributable#getPossibleValues()
 	 * 
 	 * @return {@link LabTest} object(s)
-	 * @deprecated Data provided by this method can be better achieved from 
-	 * appropriate service at point of use.
+	 * @deprecated Data provided by this method can be better achieved from appropriate service at point
```
Comment (seekme94) [inline api/src/main/java/org/openmrs/module/commonlabtest/LabTest.java:159]: Unnecessary change, breaking the formatting

File: api/src/main/java/org/openmrs/module/commonlabtest/LabTest.java [hunk 3]
```diff
@@ -220,6 +220,28 @@ public String serialize() {
...
+	 * <br>
+	 * <br>
+	 * Returns null if this commonlabtest has no non-voided {@link LabTestAttribute} with the given
+	 * {@link LabTestAttributeType}, the given {@link LabTestAttributeType} is null, or this
+	 * commonlabtest has no attributes.
+	 * 
+	 * @param lat the LabTestAttributeType to look for {@link LabTestAttributeType#equals(Object)}
+	 * @return LabTestAttribute that matches the given type
+	 */
+	public LabTestAttribute getAttribute(LabTestAttributeType lat) {
```
Comment (seekme94) [inline api/src/main/java/org/openmrs/module/commonlabtest/LabTest.java:234]: Change the parameter name to fully qualified name.

**Review by seekme94 — CHANGES_REQUESTED** [review]

File: api/src/main/java/org/openmrs/module/commonlabtest/LabTest.java (hunk 1 above)
Comment (tahiraniazi) [reply api/src/main/java/org/openmrs/module/commonlabtest/LabTest.java:135]: this is related to code format, and is a result of maven clean install

File: api/src/main/java/org/openmrs/module/commonlabtest/LabTest.java (hunk 2 above)
Comment (tahiraniazi) [reply api/src/main/java/org/openmrs/module/commonlabtest/LabTest.java:159]: this is related to code format, and is a result of maven clean install

File: api/src/main/java/org/openmrs/module/commonlabtest/LabTest.java (hunk 3 above)
Comment (tahiraniazi) [reply api/src/main/java/org/openmrs/module/commonlabtest/LabTest.java:234]: Something similar in  OpenMRS core:

https://github.com/openmrs/openmrs-core/blob/master/api/src/main/java/org/openmrs/Person.java
-- > public PersonAttribute getAttribute(PersonAttributeType pat)
